Tools/Scripts/webkitpy/__init__.py

```diff
@@ -92,8 +92,8 @@
 AutoInstall.register(Package('soupsieve', Version(2, 2, 1)))
 
 if sys.platform == 'linux':
-    AutoInstall.register(Package('selenium', Version(4, 24, 0), wheel=True, implicit_deps=[
-        Package('websocket', Version(1, 8, 0), pypi_name='websocket-client')]))
+    AutoInstall.register(Package('websocket', Version(1, 8, 0), pypi_name='websocket-client'))
+    AutoInstall.register(Package('selenium', Version(4, 24, 0), wheel=True, implicit_deps=['websocket']))
```

Comment:
I think is worth adding one-line comment here explaining why we have to do this, to alert anyone on the future modifying this lines